src/components/RouterPage.tsx

```diff
@@ -0,0 +1,15 @@
+import { LogoHeader } from "./LogoHeader";
+
+interface RouterPageProps {
+  page: JSX.Element;
+  errorMessage?: string;
+}
+
+export function RouterPage({ page, errorMessage }: RouterPageProps) {
+  return (
+    <>
+      <LogoHeader errorMessage={errorMessage} />
+      {page}
+    </>
+  );
+}
```

Comment:
puts logo and error display onto each page

Reply:
TODO: add version here

Reply:
throw a link to our ToS too

Reply:
f32c254